updates/20/updates.sql

```diff
@@ -207,4 +207,8 @@ INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xBb3A8FD6Ec4b
 INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0xd517cE9206c09dfaA7e7F40f98e59f54Fb10E09f", "Meta Merge Mana", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0xd517cE9206c09dfaA7e7F40f98e59f54Fb10E09f", "A", 56, "0xd517cE9206c09dfaA7e7F40f98e59f54Fb10E09f", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0xd517cE9206c09dfaA7e7F40f98e59f54Fb10E09f", "MMM", "meta-merge-mana", "", NULL, 1683787914, NULL);
 *
 INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xBFAbdE619ed5C4311811cF422562709710DB587d", "Diva Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xBFAbdE619ed5C4311811cF422562709710DB587d", "A", 1, "0xBFAbdE619ed5C4311811cF422562709710DB587d", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0xBFAbdE619ed5C4311811cF422562709710DB587d", "DIVA", "", "", NULL, 1687892411, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", "Wall Street Memes", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", "A", 1, "0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", "WSM", "wall-street-memes", "", NULL, 1697101248, NULL);
```

Comment:
wsm seems to be at cruptocompare https://www.cryptocompare.com/coins/wsm/overview

```suggestion
INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", "Wall Street Memes", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", "A", 1, "0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0xB62E45c3Df611dcE236A6Ddc7A493d79F9DFadEf", "WSM", "wall-street-memes", "WSM", NULL, 1697101248, NULL);
```